packages/HyperSpy.yml

```diff
@@ -0,0 +1,9 @@
+name: HyperSpy
+repo: hyperspy/hyperspy
+section: domain specific libraries
+description: Multidimensional data analysis, e.g. hyperspectral imaging.
+site: https://hyperspy.org # optional, default repo site
+keywords: [spectrum images, multidimensional data] # optional
+pypi_name: hyperspy # optional, default repo name
+conda_package: hyperspy # optional
+conda_channel: conda-forge # optional, default conda-forge
```

Comment:
You can remove the now-useless comments.

Reply:
sure, should be done now